driver-sync/src/main/com/mongodb/client/internal/MongoClientDelegate.java

```diff
@@ -54,22 +54,14 @@
 import static com.mongodb.internal.connection.ClusterDescriptionHelper.getAny;
 import static com.mongodb.internal.connection.ClusterDescriptionHelper.getAnyPrimaryOrSecondary;
 
-/**
- * This class is not part of the public API and may be removed or changed at any time.
- */
-public class MongoClientDelegate {
+final class MongoClientDelegate {
```

Comment:
Now that `com.mongodb.MongoClient` delegates fully to `com.mongodb.client.MongoClientImpl`, this class should be inlined into the latter.  I'll do it in a separate PR though.

Reply:
Ack